File: qooxdoo-contrib/QWT/branches/0.7.x/toolkit/compiler/src/test/resources/type/overloaded.java
package type;

class overloaded {
    public overloaded() {
    }
    public overloaded(int x) {
    }
    public void method(char c) {
    }
    public void method(int i) {
    }
    public static void x() {
    }
    public static void x(int a) {
    }
    public void x(Object a) {
		new overloaded();
		new overloaded(1);

        method('c');
        method(1);

        x();
        x(1);
        x(null);

    }
}

File: qooxdoo-contrib/QWT/tags/0.7.2/toolkit/compiler/src/test/resources/type/overloaded.js
touchPackage("type");
type.overloaded=function() {
}
defineClass(type.overloaded, 'type.overloaded', java.lang.Object);
type.overloaded.init = function() {
}
type.overloaded.init1 = function(x) {
}
type.overloaded.prototype.method = function(c) {
}
type.overloaded.prototype.method1 = function(i) {
}
type.overloaded.x = function() {
}
type.overloaded.x1 = function(a) {
}
type.overloaded.prototype.x2 = function(a) {
    newObject(type.overloaded, type.overloaded.init, []);
    newObject(type.overloaded, type.overloaded.init1, [1]);
    this.method(99);
    this.method1(1);
    type.overloaded.x();
    type.overloaded.x1(1);
    this.x2(null);
}
